paasta_tools/contrib/paasta_update_soa_memcpu.py

```diff
@@ -1,52 +1,44 @@
 #!/usr/bin/env python3
 import argparse
 import contextlib
+import json
 import os
 import subprocess
-import sys
 import tempfile
+import time
 
+import requests
 import ruamel.yaml as yaml
+from jira.client import JIRA
 
 
 def parse_args(argv):
     parser = argparse.ArgumentParser(
         description='',
     )
     parser.add_argument(
-        '-s', '--service',
-        help="Service to edit. Like 'example_service'.",
-        dest="service",
+        '-s', '--splunk_creds',
+        help="Creds for Splunk API, user:pass",
+        dest="splunk_creds",
         required=True,
     )
     parser.add_argument(
-        '-i', '--instance',
-        help="Instance of the service to edit. Like 'main' or 'canary'.",
-        dest="instance",
+        '-j', '--jira_creds',
+        help="Creds for JIRA API, user:pass",
+        dest="jira_creds",
         required=True,
     )
     parser.add_argument(
-        '-c', '--cluster',
-        help="The PaaSTA cluster that has the service instance to edit. Like 'norcal-prod'.",
-        dest="cluster",
-        required=True,
-    )
-    parser.add_argument(
-        '-m', '--mem',
-        default='',
-        dest="mem",
-        help='New value for mem of service',
+        '-n', '--no-tick',
+        help='Do not create a JIRA ticket',
+        action='store_true',
+        dest='no_tick',
+        default=False,
     )
     parser.add_argument(
-        '-cp', '--cpu',
-        default='',
-        dest="cpu",
-        help='New value for cpus of service',
-    )
-    parser.add_argument(
-        '-t', '--ticket',
-        dest="ticket",
-        help='JIRA ticket tracking changes',
+        '-f', '--file-splunk',
+        help='Splunk csv from which to pull data. Defaults to paasta_overprovision_alerts_fired.csv',
```

Comment:
I don't see where we defined this default. The jenkins job at least it will be always be provided, I'm not sure a default is needed (except to help humans).